### ISSUE TITLE
The target box does not tightly surround the target

### ISSUE THREAD

**Issue by GKYoffice** [ultralytics/ultralytics#18448]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

![image](https://github.com/user-attachments/assets/9d395751-53f2-4d6b-884a-696944f686b4)
![image](https://github.com/user-attachments/assets/9f68a834-b0ef-4f34-9fe4-67bc01f4906b)
Why can't my target box tightly surround my target, especially when the target is small? Does this depend on the size of the feature map

What I expected:
![image](https://github.com/user-attachments/assets/97a33bee-a89b-4b02-81da-b23aa1643aaf)

Thanks！

### Additional

no

**Comment by UltralyticsAssistant** [comment]
👋 Hello @GKYoffice, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com) for guidance, where you can find helpful [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. Many common questions are addressed there.

If this is a 🐛 Bug Report, it would greatly help if you could provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us debug it effectively. For example, share relevant code snippets, configurations, image samples, or additional details that clearly illustrate the issue.

If this is a ❓ Question about custom training or model performance, please include more information, such as:
- Your training hyperparameters and settings
- Details about your dataset (e.g., class distribution, image resolution)
- Examples of your predictions and logs
Be sure to follow our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

### Upgrade
To ensure this issue isn’t already addressed in the latest version, ensure your `ultralytics` package and dependencies are up to date. Upgrade in a [**Python>=3.8**](https://www.python.org/) environment with:

```bash
pip install -U ultralytics
```

### Environments
YOLO runs seamlessly in the following verified environments, where dependencies like [CUDA](https://developer.nvidia.com/cuda), [CUDNN](https://developer.nvidia.com/cudnn), and [PyTorch](https://pytorch.org/) are pre-installed:
- **Notebooks** with free GPUs: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Run in Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

### Community
Join our awesome Ultralytics community:
- Chat live on [Discord](https://discord.com/invite/ultralytics) 🎧
- Share insights on [Discourse](https://community.ultralytics.com)
- Dive into discussions on our [Subreddit](https://reddit.com/r/Ultralytics)

### Status
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

Our CI tests verify YOLO’s correctness on various platforms daily. If this badge is green, all [CI tests](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) are passing ✅.

This is an automated response to get you started on resolving your issue. An Ultralytics engineer will review and respond further soon. Thanks for your patience and happy coding! 😊

**Comment by Y-T-G** [comment]
You can try training with larger imgsz
